README.md

```diff
@@ -1,4 +1,8 @@
 # react-mapbox-gl-draw
+
+This is fork of [react-mapbox-gl-draw](https://github.com/amaurymartiny/react-mapbox-gl-draw)
+that implements draw events.
+
```

Comment:
Can you remove this part of the readme? After merge it won't be a fork anymore.

Reply:
alright